internal/auth/configuration.go

```diff
@@ -84,6 +90,9 @@ func DefaultAuthConfig() Configuration {
 				Secure:   true,
 				HTTPOnly: true,
 			},
+			RedisConfig: RedisConfig{
+				UseSentinel: false,
```

Comment:
Yes, that's correct. It's been awhile but I think it might fail if the `RedisConfig` struct doesn't get initialized, but the `UseSentinel` default is the `false` zero value.